### ISSUE TITLE
get All people match wrong organisation

### ISSUE THREAD

**Issue by SimonLab** [postgres-plugin/people#53]
the [getAllPeople query](https://github.com/postgres-plugin/people/blob/master/lib/queries/getAllPeople.js) return a name organisation for each people even when the people is not linked to an organisation. @JMurphyWeb @mk4111  is this due to the "left join"?

**Comment by SimonLab** [comment]
The query should join on people.org_id instead of people.id with the organisation
```
LEFT JOIN organisations ON (people.org_id = organisations.id)
```

**Comment by jay-meister** [comment]
well spotted @SimonLab

**Comment by jay-meister** [comment]
resolved with #54
